gatsby-config.js

```diff
@@ -0,0 +1,10 @@
+/**
```

Comment:
@ORKO06 did you check the config for oregano. There are quite a few things that need to be configured